recipes/poco/all/patches/1.9.0.patch

```diff
@@ -0,0 +1,58 @@
+--- cmake/PocoMacros.cmake
++++ cmake/PocoMacros.cmake
+@@ -265,7 +265,7 @@
+ 
+ if (MSVC)
+ # install the targets pdb
+-  POCO_INSTALL_PDB(${target_name})
++  # POCO_INSTALL_PDB(${target_name})
+ endif()
+ 
+ endmacro()
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -211,7 +211,7 @@
+ 
+ find_package(OpenSSL)
+ if(OPENSSL_FOUND)
+-    include_directories("${OPENSSL_INCLUDE_DIR}")
++    include_directories("${OpenSSL_INCLUDE_DIR}")
```

Comment:
Using targets, shouldn't be needed

Reply:
Just avoiding this error because of missing variables in generated find_package.
```
CMake Error at source_subfolder/CMakeLists.txt:207 (include_directories):
  include_directories given empty-string as include directory
```

Reply:
Same comment above